app/models/food.rb

```diff
@@ -0,0 +1,11 @@
+class Food < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
+  # belongs_to :recipe
+  has_many :recipe_foods, dependent: :destroy
+
+  validates :name, presence: true, uniqueness: true, length: { minimum: 3, maximum: 50 }
+  validates :measurement_unit, presence: true, length: { minimum: 2, maximum: 10 }
+  validates :price, presence: true, numericality: { greater_than: 0, less_than: BigDecimal(10**3) },
+                    format: { with: /\A\d+(?:\.\d{0,2})?\z/ }
+  validates :quantity, presence: true, numericality: { greater_than: 0, less_than: 10**3 }
+end
```

Comment:
**OPTIONAL** - You have done well using validations in your project, but some of your validations are strict, and since you are not displaying a flash message to notify the user what might be wrong, this can leave a frustrating feeling for the user. I suggest you make some of your validations especially for the price and quantity a little easier. Please fix this 🙏